content/en/blog/falcoctl-install-manage-rules-plugins.md/index.md

```diff
@@ -0,0 +1,383 @@
+---
+title: "Falcoctl: The basic usages"
+linktitle: "Falcoctl: The basic usages"
+date: 2023-02-16
+author: Thomas Labarussias
+slug: falcoctl-basic-usages
+---
+
+Since the launch of the plugin framework in January 2022, our adopters have requested an out-of-the-box solution to manage the lifecycle of rules (installation, updates), and a how to guide to smoothly install the [plugins](https://falco.org/docs/plugins/).
```

Comment:
```suggestion
Since the launch of the plugin framework in January 2022, our adopters have requested an out-of-the-box solution to manage the lifecycle of rules (installation, updates). We heard your request and also created a guide to help you smoothly install the [plugins](https://falco.org/docs/plugins/).
```